libvast/vast/config.hpp.in

```diff
@@ -13,6 +13,7 @@
 
 #include <caf/config.hpp>
 
+#define VAST_VERSION_LONG "@VAST_VERSION@"
 #define VAST_VERSION "@VERSION_MAJ_MIN@"
```

Comment:
Is this used anywhere after you migrated the version command to `_LONG`? If not, I would suggest to drop the `_LONG` suffix and simply make `VAST_VERSION` be the long version by default.

Reply:
I changed it accordingly and also did some cleanup.